lib/publiccloud/instance.pm

```diff
@@ -264,6 +264,36 @@ sub upload_log {
     assert_script_run("test -d '$tmpdir' && rm -rf '$tmpdir'");
 }
 
+=head2 verify_upload_logs_from_dir
+
+    verify_upload_logs_from_dir($dir, @files, $log_header);
+
+Check files status and upload logs ok to UI. This works on files in a specific directory.
+Input: C<$dir> directory path, C<@files> files basenames array; C<$log_header> the leading name added to each log
+
+For more directories to manage, repeated calls for each one can be done.
+Still, you can also set empty dir "" and use full path names, from root.
+=cut
+
+sub verify_upload_logs_from_dir {
+    my ($self, $dir, @files, $log_header) = @_;
```

Comment:
I don't think that `$log_header` works over here as the follows after an expected array parameter

Reply:
do you mean should be placed before the array? I was tempted to add hash form {a => x} but is too much code added.

Reply:
yes, I think. Anyway, the log header is not really indeed IMHO. I would not break the convention that uploaded assets start with the module name by which they were uploaded.

Reply:
Header input removed, standard module name applied